dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -69,13 +69,6 @@
         return null;
     }
 
-    @Override
-    public T findOne(ID id) {
-        Context context = obtainContext();
-        return findOne(context, id);
```

Comment:
The Spring Security annotations are based on Spring AOP. Spring AOP will enhance each annotated class with a facade. This facade will check each incoming method call from another object for the correct authorizations. However, method calls within the same class are not checked.

Since `org.dspace.app.rest.RestResourceController`always uses `findOne(ID id)` without `Context`, the only method that can be annotated is `findOne(ID id)`. Therefore we removed it from the super-class so that each sub-class can define his own custom permissions.

So an alternative approach to change `RestResourceController` to use `findOne(Context context, ID id)`. 

However, it is important that we only keep either `findOne(ID id)` **or** `findOne(Context context, ID id)` for the annotations to work.